Makefile.am

```diff
@@ -12,7 +12,7 @@ endif
 ACLOCAL_AMFLAGS = -I m4
 EXTRA_DIST = doxygen.cfg
 library_includedir=$(includedir)/usbg
-library_include_HEADERS = include/usbg/usbg.h
+library_include_HEADERS = include/usbg/usbg.h include/usbg/usbg_version.h
```

Comment:
You add a new header so you will need to also update spec file (packaging/libusbgx.spec)